src/routes/target/target.spec.ts

```diff
@@ -1,6 +1,17 @@
 import { handler as target } from './index';
 import { Request, Response } from 'express';
 
+jest.mock('../health', () => ({
```

Comment:
This mock is repeated in eahc of the routes. If you put a `__mocks__` directory adjacent to `src/routes/health/index.ts` and put that mock inside, `jest` will automatically imported that mocked module during our tests.

Reply:
This has been resolved in the latest changes.